### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by jdrestre**
Hello Gio. Update methods into engines storage. Include Methods `get` and `count`. I review pep8 in both files and it is OK. Please review if methds function in your machine and tell me.
The follow screencapture show you the methods added
image

### CODE REVIEW

File: models/engine/file_storage.py [hunk 1]
```diff
@@ -68,3 +68,19 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Retrieve one object"""
+        data = self.all(cls)
+        for k, v in data.items():
```
Comment (Bard-Budist) [inline models/engine/file_storage.py]: Why you don't use the same  loop for this file? ` for k, v in data.items():`

**Review by Bard-Budist — CHANGES_REQUESTED** [review]
Please use de same struct in loop, in this case in the file` file_storage.py`

File: models/engine/file_storage.py (hunk 1 above)
Comment (jdrestre) [reply models/engine/file_storage.py]: Ok Perfect, I forgot update this change into file_storage engine.

**Review by Bard-Budist — APPROVED** [review]
Nice, beautiful!

**Pull request merged**